### ISSUE TITLE
File ownership

### ISSUE THREAD

**Issue by lopippo** [mhoopmann/mstoolkit#13]
Greetings, Michael,

The file  mstoolkit/src/mzParser/mzp_base64.cpp really looks like it is based on the work here:
https://github.com/PedrioliLab/ReAdW/blob/master/src/mzXML/common/Base64.cpp

Might it be acceptable to adopt that file or to say in the license stanza, instead of "downloaded from web"  that it is based on that file (by definition anything published anywhere is "all rights reserved" if nothing is granted, Bern convention) ? In that case, no need to change the code, just acknowledge that the file is based upon the work of others ? Of course, only if my initial assumption is correct.

Thank you for your help.
Cheers,
Filippo <lopippo@debian.org>

**Comment by mhoopmann** [comment]
No, that "downloaded from web" portion is part of the original file. Your suggested source is not where the file originated from. I obtained Base64.cpp from here: ftp://ftp.thegpm.org/repos/tandem/ which is under artistic license. Although I haven't changed whatever copyright that was included in the file, I've now extended the information to include the copyright and license of the source package the file was obtained from.
Cheers,
Mike
